tests/test_atom_mapping.py

```diff
@@ -1,15 +1,34 @@
-from timemachine.fe.atom_mapping import get_core_by_mcs, mcs_map
+from rdkit import Chem
+
+from timemachine.fe.atom_mapping import get_core_by_mcs, mcs
 from timemachine.testsystems.relative import hif2a_ligand_pair
 
 
 def test_mcs_map():
-    mcs_result = mcs_map(hif2a_ligand_pair.mol_a, hif2a_ligand_pair.mol_b, threshold=0.5)
+    mcs_result = mcs(hif2a_ligand_pair.mol_a, hif2a_ligand_pair.mol_b, threshold=0.5)
```

Comment:
nit: update test name?

Reply:
Ahh, good call -- fixed in https://github.com/proteneer/timemachine/pull/812/commits/23aa25ee24ca8368f63cedb91c1716a15b3fee8c